sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_usage_v2/schema.yaml

```diff
@@ -124,3 +124,8 @@ fields:
   name: resource_warning
   type: STRING
   description: The warning message that appears if the resource usage is above the internal threshold of the system
+
+- mode: NULLABLE
+  name: submission_date
+  type: DATE
+  description: Date bigquery_usage_v2 table refreshed
```

Comment:
Is this actually when the table was refreshed or is this date corresponding to the Airflow's execution date?

Reply:
execution date - I was trying to figure out a more descriptive way. But I'll change this comment